src/Raven.Server/Web/Authentication/AdminCertificatesHandler.cs

```diff
@@ -880,123 +880,149 @@ public Task RenewalDate()
             return Task.CompletedTask;
         }
 
+        [RavenAction("/admin/certificates/letsencrypt/force-renew", "OPTIONS", AuthorizationStatus.ClusterAdmin)]
+        [RavenAction("/admin/certificates/refresh", "OPTIONS", AuthorizationStatus.ClusterAdmin)]
+        [RavenAction("/admin/certificates/replace-cluster-cert", "OPTIONS", AuthorizationStatus.ClusterAdmin)]
+
```

Comment:
Why is this method handling the `OPTIONS` here?

Reply:
Sorry, I didn't copy the actual handler.